src/NuGet.Status/Web.config

```diff
@@ -1,172 +1,94 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0"?>
 <!--
   For more information on how to configure your ASP.NET application, please visit
   http://go.microsoft.com/fwlink/?LinkId=301880
   -->
 <configuration>
   <appSettings>
-    <add key="webpages:Version" value="3.0.0.0" />
-    <add key="webpages:Enabled" value="false" />
-    <add key="ClientValidationEnabled" value="true" />
-    <add key="UnobtrusiveJavaScriptEnabled" value="true" />
-
+    <add key="webpages:Version" value="3.0.0.0"/>
+    <add key="webpages:Enabled" value="false"/>
+    <add key="ClientValidationEnabled" value="true"/>
+    <add key="UnobtrusiveJavaScriptEnabled" value="true"/>
     <!--Settings-->
-    <add key="Storage:ConnectionString" value="UseDevelopmentStorage=true" />
-    <add key="Storage:SecondaryConnectionString" value="UseDevelopmentStorage=true" />
-    <add key="Storage:ContainerName" value="status" />
-    <add key="Storage:BlobName" value="status.json" />
-    <add key="Storage:TableName" value="status" />
-
-    <add key="KeyVault:VaultName" value="" />
-    <add key="KeyVault:CertificateThumbprint" value="" />
-    <add key="KeyVault:ClientId" value="" />
-    <add key="KeyVault:StoreLocation" value="" />
-    <add key="KeyVault:StoreName" value="" />
-    <add key="KeyVault:ValidateCertificate" value="false" />
-    
-    <add key="NuGetBaseUrl" value="https://dev.nugettest.org" />
-    
-    <add key="Ida:AdminEnabled" value="false" />
-    <add key="Ida:RootUri" value="" />
-    <add key="Ida:AADInstance" value="" />
-    <add key="Ida:ClientId" value="" />
-    <add key="Ida:Tenant" value="" />
-    
-    <add key="ExternalBrandingMessage" value="© Microsoft 2019" />
-    <add key="ExternalPrivacyPolicyUrl" value="https://go.microsoft.com/fwlink/?LinkId=521839" />
-    <add key="TrademarksUrl" value="https://www.microsoft.com/trademarks" />
-
-    <add key="ApplicationInsightsKey" value="" />
+    <add key="Storage:ConnectionString" value="UseDevelopmentStorage=true"/>
+    <add key="Storage:SecondaryConnectionString" value="UseDevelopmentStorage=true"/>
+    <add key="Storage:ContainerName" value="status"/>
+    <add key="Storage:BlobName" value="status.json"/>
+    <add key="Storage:TableName" value="status"/>
+    <add key="KeyVault:VaultName" value=""/>
+    <add key="KeyVault:CertificateThumbprint" value=""/>
+    <add key="KeyVault:ClientId" value=""/>
+    <add key="KeyVault:StoreLocation" value=""/>
+    <add key="KeyVault:StoreName" value=""/>
+    <add key="KeyVault:ValidateCertificate" value="false"/>
```

Comment:
Visual Studio auto-formatted this. I can revert if you want but I figured it's best to just align with what the tooling wants.